pkg/common/strutils/strutils.go

```diff
@@ -1,6 +1,18 @@
 package strutils
 
+import "errors"
+
 // Sringp returns a string pointer of the given string
 func Stringp(str string) *string {
 	return &str
 }
+
+// AllSet checks whether all provided string values are non-empty
+func AllSet(vals ...string) error {
```

Comment:
if one of the strings is empty it is not really an "error" but just a regular result of the purpose of this func.
I think we should return `bool` instead of `error`